### PR TITLE
Add guides to navbar

### PR DESCRIPTION
**Pull request by Gayathri625**
Improving discoverability of guides page by adding guides to the navbar menu

### CODE REVIEW

**Comment by arjan-bal** [comment]
To be reviewed by @temawi who has context about these changes.

**Review by arjan-bal — APPROVED** [review]
nit: Consider change the PR title to "Add guides to navbar" instead. Otherwise LGTM.

**Review by gnossen — APPROVED** [review]

File: content/en/docs/guides/_index.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Guides
 description: Task-oriented walkthroughs of common use cases
 weight: 4
 nav_children: pages
+menu:
+  main: {weight: 3}
 ---
```
Comment (Muzaffer8217) [inline content/en/docs/guides/_index.md:9]: muzafferbayindir015@gmail.com
Muzaffer Bayındır 
TÜRKİYE-ANTALYA 
+905327354707

Comment (Muzaffer8217) [reply content/en/docs/guides/_index.md:9]: > muzafferbayindir015@gmail.com
> Muzaffer Bayındır 
> TÜRKİYE-ANTALYA 
> +905327354707

**Pull request merged**